css/style.css

```diff
@@ -1,11 +1,261 @@
+    padding: 10px 20px;
+}
+
+.header .menu {
+    clear: both;
+    max-height: 0;
+    transition: max-height .2s ease-out;
+}
+
+.header .menu-icon {
+    border: 1px solid #838994;
+    padding: 28px 20px;
+    position: relative;
+    float: right;
+    cursor: pointer;
+}
+
+.header .menu-icon .nav-icon {
+    background-color: #838994;
+    display: block;
+    height: 2px;
+    width: 18px;
+    position: relative;
+    transition: background .2s ease-out;
+}
+
+.header .menu-icon .nav-icon:before {
+    background: #838994;
+    content: "";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: 5px;
+
+}
+
+.header .menu-icon .nav-icon:after {
+    background: #838994;
+    content: "";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: -5px;
+}
+
+.header .menu-btn {
+    display: none;
+}
+
+.header .menu-btn:checked~.menu {
+    max-height: 240px;
+}
+
+.header .menu-btn:checked~.menu-icon .nav-icon {
+    background-color: transparent;
+}
+
+.header .menu-btn:checked~.menu-icon .nav-icon:before {
+    transform: rotate(-45deg);
+    top: 0;
+}
+
+.header .menu-btn:checked~.menu-icon .nav-icon:after {
+    transform: rotate(45deg);
+    top: 0;
+}
+
+@media (min-width:48em) {
+
+    .header li {
+        float: left;
+
+    }
+
+    .header li a {
+        padding: 20px 30px;
+    }
+
+    .header .menu {
+        clear: none;
+        float: right;
+        max-height: none;
+    }
+
+    .header .menu-icon {
+        display: none;
+    }
+}
+
+.hero-picture {
+    background-image: url("../img/first-background.jpg");
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 45.938rem;
+    overflow: auto;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+}
+
+.hero-content {
+    color: white;
+    display: flex;
+    flex-direction: column;
+}
+
+.hero-content h1 {
+    text-align: center;
+    font-size: 50px;
+    font-weight: 300;
+    margin: 10px;
+}
+
+.hero-content h3 {
+    text-align: center;
+    font-size: 25px;
+    font-weight: 300;
+    margin: 10px;
+}
+
+
+.button {
+    background-color: #f15a29;
+    border-color: #f15a29;
+    padding: 15px;
+    border-radius: 4px;
+    width: 140px;
+    margin-top: 20px;
+    font-size: 18px;
+    text-align: center;
+}
+
+section div div {
+    text-align: center;
+}
+
+/* second section */
+
+.second-section-h1 {
+    font-weight: 300;
+    margin-top: 100px;
+    text-align: center;
+    font-size: 45px;
+}
+
+section div>img {
+    width: 127px;
+    height: 127px;
+}
+
+.flex-container {
+    flex-direction: row;
+    display: flex;
+    justify-content: center;
+    margin-top: 60px;
+}
+
+@media (max-width: 650px) {
+    .flex-container {
+        flex-direction: column;
+    }
+}
+
+section div div {
+    margin: 20px;
```

Comment:
Watch out for rules that are applied to generic elements such as `div` or `section` - you may not _always_ want the margin on all divs for example.